app/policies/user_policy.rb

```diff
@@ -21,6 +21,6 @@ def update_settings?
 
   # Only users that are instructors in at least one course
   def reset_api_key?
-    user.roles.pluck(:type).include?('Instructor')
+    user.roles.pluck(:type).include?('Instructor') || user.roles.pluck(:type).include?('Student')
```

Comment:
You can make this more general and allow all `StandardUser`s to reset their API key. Also remember to update the comment above